<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentStatus</class>
 <widget class="QDialog" name="DocumentStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="docLbl">
     <property name="text">
      <string>Формирование документов</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="docStat">
     <property name="text">
      <string>Выполняется...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="closeCheck">
     <property name="text">
      <string>Закрыть после завершения </string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="closeBtn">
     <property name="toolTip">
      <string>Выход</string>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
